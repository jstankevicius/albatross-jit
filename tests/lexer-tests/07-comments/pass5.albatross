# Below is a mix of passing tokens sometimes commented and bad tokens always commented
3935388	9535985	4888807	9232923	9602237	3251510	1680588	3825847	6960825 364609 1068096	2904711	2829862	2553238	7451660	5573677	2378447	8550385	9657404 7599672
#6915841	1201399	9800537	9760121	1469487	1728781	7034077	9537836	36915665990957
2752823	5904122	5543518	7744084	5520173	4914176	5207690	254915	9373537 6780194 8871163	8960451	2718878	6083811	9031148	1846729	9506452	3198971	301950 3824649
#4369575	8373847	6261978	2177314	8149875	4293729	2761083	4027483	99682284296303968228429630396822842963039682284296303968228429630396822842963036822842963039682284296303
#86652c9	1413224	4950027	754528	8643914	6997321	6567891	5187967	66886485190075468864851900754688648519007546886485190075468864851900754688648519007548864851900754
