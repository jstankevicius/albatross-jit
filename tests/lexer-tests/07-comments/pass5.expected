1 2 INT 3935388
9 2 INT 9535985
17 2 INT 4888807
25 2 INT 9232923
33 2 INT 9602237
41 2 INT 3251510
49 2 INT 1680588
57 2 INT 3825847
65 2 INT 6960825
73 2 INT 364609
80 2 INT 1068096
88 2 INT 2904711
96 2 INT 2829862
104 2 INT 2553238
112 2 INT 7451660
120 2 INT 5573677
128 2 INT 2378447
136 2 INT 8550385
144 2 INT 9657404
152 2 INT 7599672
1 4 INT 2752823
9 4 INT 5904122
17 4 INT 5543518
25 4 INT 7744084
33 4 INT 5520173
41 4 INT 4914176
49 4 INT 5207690
57 4 INT 254915
64 4 INT 9373537
72 4 INT 6780194
80 4 INT 8871163
88 4 INT 8960451
96 4 INT 2718878
104 4 INT 6083811
112 4 INT 9031148
120 4 INT 1846729
128 4 INT 9506452
136 4 INT 3198971
144 4 INT 301950
151 4 INT 3824649
